alchemist-incarnation-collektive/src/main/kotlin/it/unibo/collektive/alchemist/device/sensors/LocalSensing.kt

```diff
@@ -0,0 +1,11 @@
+package it.unibo.collektive.alchemist.device.sensors
+
+/**
+ * Distance sensor.
+ */
+interface LocalSensing {
+    /**
+     * Sense a molecule given its [name] and returns its value.
+     */
+    fun <T> sense(name: String): T
```

Comment:
Sometimes, it can be useful to have a default value if the molecule does not exist.

I suggest adding `senseOrElse` with a default value if the molecule does not exist.